src/custom/components/Button/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import styled from 'styled-components'
+import { ButtonProps } from 'rebass/styled-components'
+import { ChevronDown } from 'react-feather'
+
+import { RowBetween } from 'components/Row'
+
+import {
+  // Import only the basic buttons
+  ButtonPrimary as ButtonPrimaryMod,
+  ButtonLight as ButtonLightMod,
+  ButtonGray as ButtonGrayMod,
+  ButtonSecondary as ButtonSecondaryMod,
+  ButtonPink as ButtonPinkMod,
+  ButtonOutlined as ButtonOutlinedMod,
+  ButtonEmpty as ButtonEmptyMod,
+  ButtonWhite as ButtonWhiteMod,
+  ButtonConfirmedStyle as ButtonConfirmedStyleMod,
+  ButtonErrorStyle as ButtonErrorStyleMod
+  // We don't import the "composite" buttons, they are just redefined (c&p actually)
+} from './ButtonMod'
+
+export const ButtonPrimary = styled(ButtonPrimaryMod)`
+  // CSS overrides
+  background-color: transparent;
+  background-image: ${({ theme }) => `linear-gradient(270deg, ${theme.purple} 30%, ${theme.blue1} 70%);`};
+  border: 0;
+
+  &:focus,
+  &:hover,
+  &:active {
+    background-color: transparent;
+    background-image: ${({ theme }) => `linear-gradient(270deg, ${theme.purple} 30%, ${theme.blue1} 70%);`};
```

Comment:
Tried making the code dryer by putting this line into a variable. It's also used on line 26. Need help refactoring as the gradient will be likely re-used more and more. Perhaps it makes sense to define this variable in the general theme file instead.

Reply:
I see 2 options: 
- https://github.com/gnosis/dex-swap/pull/20